compiler/circle-execution-plan/src/CircleExecutionPlan.cpp

```diff
@@ -49,6 +61,33 @@ int entry(int argc, char **argv)
 
   std::string input_path = arser.get<std::string>("input");
   std::string output_path = arser.get<std::string>("output");
+  std::string platform_name = arser.get<std::string>("--platform");
+  bool use_dsp = arser.get<bool>("--use_dsp");
```

Comment:
(optional) Personally, I prefer use `const` as much as possible because it helps to reviewer not expect variable changes bellow of definition. Hovewer this comment is optional because some variables above is not `const` too.